Public/Src/Demos/Deployment.dsc

```diff
@@ -14,7 +14,7 @@ namespace Deployment {
         ]
     };
 
-    const frameworkSpecificPart = qualifier.targetFramework === "netcoreapp2.2"
+    const frameworkSpecificPart = qualifier.targetFramework === "netcoreapp3.0"

```

Comment:
>ier.targetFramework === "netcoreapp [](start = 40, length = 35)

`BuildXLSdk.isNetCoreApp` or `!BuildXLSdk.isFullFramework`

Reply:
Note, will not comment on each and every one :)
Note not asking you to fix before merging but if you don't fix it let me know as I want to fix it :)

---
In reply to: [283563371](https://github.com/microsoft/BuildXL/pull/298#discussion_r283563371) [](ancestors = 283563371)

Reply:
Fixed.